core/src/test/java/nl/nn/adapterframework/xml/XmlPrettyPrintFilterTest.java

```diff
@@ -47,8 +47,8 @@ public void testNoLexicalHandling() throws Exception {
 		filter.setContentHandler(xmlWriter);
 
 		InputSource inputSource = new InputSource(new StringReader(input));
-		XMLReader xmlReader = XmlUtils.getXMLReader(true, true);
-		xmlReader.setContentHandler(filter);
+		XMLReader xmlReader = XmlUtils.getXMLReader(filter);
+		xmlReader.setProperty("http://xml.org/sax/properties/lexical-handler", null);
```

Comment:
Maybe you could add a small description (comment) why you're explicitly turning off the lexical-handler